### PR TITLE
LLM: add dolly-v1 and dolly-v2 to gpu pytorch model example

### PR DESCRIPTION
**Pull request by JinBridger**
## Description

Add dolly-v1 and dolly-v2 to GPU PyTorch model example.

### CODE REVIEW

File: python/llm/example/GPU/PyTorch-Models/Model/dolly-v1/generate.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                        help='Prompt to infer')
+    parser.add_argument('--n-predict', type=int, default=32,
+                        help='Max tokens to predict')
+
+    args = parser.parse_args()
+    model_path = args.repo_id_or_model_path
+
+    # Load model
+    model = AutoModelForCausalLM.from_pretrained(model_path,
+                                      trust_remote_code=True,
```
Comment (plusbang) [inline python/llm/example/GPU/PyTorch-Models/Model/dolly-v1/generate.py:50]: add some blank spaces to align

**Review by plusbang — COMMENTED** [review]
Add links in https://github.com/intel-analytics/BigDL/tree/main/python/llm/example/GPU/PyTorch-Models/Model#verified-models

File: python/llm/example/GPU/PyTorch-Models/Model/dolly-v1/generate.py (hunk 1 above)
Comment (JinBridger) [reply python/llm/example/GPU/PyTorch-Models/Model/dolly-v1/generate.py]: done

**Review by plusbang — APPROVED** [review]
LGTM

**Pull request merged**